apis/topology/v1beta1/conditions.go

```diff
@@ -0,0 +1,34 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	condition "github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+)
+
+
+// Common Messages used by API objects when a Topology is referenced
+const (
+	// TopologyConfigReadyInitMessage
+	TopologyConfigReadyInitMessage = "Topology config create not started"
```

Comment:
@stuggi is this good? This way we can consume these `Topology` conditions from the service operators and we don't have to redefine them every time.

Reply:
either here, or we add it to https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/condition/conditions.go . there we also have shared conditions. adding it there has the advantage that it is already a known place and we also have there conditions for memcached or transporturl , they are not part of the api which manage it. adding it here has also benefits to be able to update it with a new change to the topoligy. to keep it in sync with what we did in the past lib-common might probably be the right place.